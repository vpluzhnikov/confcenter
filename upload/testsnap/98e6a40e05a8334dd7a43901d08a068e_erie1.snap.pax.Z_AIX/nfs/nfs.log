Checking space requirement for NFS information...

Gathering NFS information... done.
