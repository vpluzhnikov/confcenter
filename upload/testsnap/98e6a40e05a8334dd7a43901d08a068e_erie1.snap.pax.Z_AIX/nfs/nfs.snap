
.....
.....     Creation Date
.....

Tue Nov 27 15:17:59 MSK 2012

.....
.....     Note: If you want the /var/adm/ras/trcfile, snap -g will get it
.....


.....
.....     lssrc -a
.....

Subsystem         Group            PID          Status 
 platform_agent                    657002       active
 cimsys                            793234       active
 snmpd            tcpip            1051918      active
 syslogd          ras              854412       active
 sendmail         mail             657224       active
 portmap          portmap          922016       active
 inetd            tcpip            790996       active
 snmpmibd         tcpip            794002       active
 xntpd            tcpip            921472       active
 hostmibd         tcpip            857482       active
 aixmibd          tcpip            922830       active
 nimsh            nimclient        989726       active
 biod             nfs              3276858      active
 rpc.statd        nfs              1051172      active
 rpc.lockd        nfs              984340       active
 qdaemon          spooler          853004       active
 writesrv         spooler          990730       active
 clcomd           caa              790414       active
 sshd             ssh              726876       active
 pconsole         pconsole         1051738      active
 ctrmc            rsct             790220       active
 IBM.ServiceRM    rsct_rm          1180844      active
 IBM.DRM          rsct_rm          792128       active
 lpd              spooler                       inoperative
 keyserv          keyserv                       inoperative
 ypbind           yp                            inoperative
 llbd             iforncs                       inoperative
 glbd             iforncs                       inoperative
 cdromd                                         inoperative
 muxatmd          tcpip                         inoperative
 rwhod            tcpip                         inoperative
 dpid2            tcpip                         inoperative
 dhcpcd           tcpip                         inoperative
 dhcpcd6          tcpip                         inoperative
 ndpd-host        tcpip                         inoperative
 ndpd-router      tcpip                         inoperative
 tftpd            tcpip                         inoperative
 netcd            netcd                         inoperative
 gated            tcpip                         inoperative
 named            tcpip                         inoperative
 routed           tcpip                         inoperative
 iptrace          tcpip                         inoperative
 timed            tcpip                         inoperative
 dhcpsd           tcpip                         inoperative
 dhcpsdv6         tcpip                         inoperative
 dhcprd           tcpip                         inoperative
 mrouted          tcpip                         inoperative
 rsvpd            qos                           inoperative
 policyd          qos                           inoperative
 pxed             tcpip                         inoperative
 binld            tcpip                         inoperative
 dfpd             tcpip                         inoperative
 nfsd             nfs                           inoperative
 rpc.mountd       nfs                           inoperative
 automountd       autofs                        inoperative
 nfsrgyd          nfs                           inoperative
 gssd             nfs                           inoperative
 cpsd             ike                           inoperative
 tmd              ike                           inoperative
 isakmpd                                        inoperative
 ikev2d                                         inoperative
 iked             ike                           inoperative
 cld              caa                           inoperative
 clconfd          caa                           inoperative
 pconsoleProxy    pconsole                      inoperative
 pconsolePDA      pconsole                      inoperative
 ctcas            rsct                          inoperative
 gsclvmd                                        inoperative
 dtsrc                                          inoperative
 IBM.ERRM         rsct_rm                       inoperative
 IBM.AuditRM      rsct_rm                       inoperative
 IBM.HostRM       rsct_rm                       inoperative
 IBM.MgmtDomainRM rsct_rm                       inoperative

.....
.....     rpcinfo -p
.....

   program vers proto   port  service
    100000    4   udp    111  portmapper
    100000    3   udp    111  portmapper
    100000    2   udp    111  portmapper
    100000    4   tcp    111  portmapper
    100000    3   tcp    111  portmapper
    100000    2   tcp    111  portmapper
    100068    2   udp   9008  cmsd
    100068    3   udp   9008  cmsd
    100068    4   udp   9008  cmsd
    100068    5   udp   9008  cmsd
    100083    1   tcp  13538  ttdbserver
    100024    1   tcp  57950  status
    100024    1   udp   9022  status
    100021    1   udp   9028  nlockmgr
    100021    2   udp   9028  nlockmgr
    100133    1   tcp  57950
    100021    3   udp   9028  nlockmgr
    100021    4   udp   9028  nlockmgr
    100021    1   tcp  14035  nlockmgr
    100021    2   tcp  14035  nlockmgr
    100021    3   tcp  14035  nlockmgr
    100021    4   tcp  14035  nlockmgr
    100133    1   udp   9030
    200001    1   tcp  57950
    200001    1   udp   9037
    200001    2   tcp  57950
    200001    2   udp   9044

.....
.....     lsnfsexp 
.....


.....
.....     exportfs
.....

exportfs: nothing exported

.....
.....     lsnfsmnt
.....

Name            Nodename   Mount Pt               VFS   Size    Options    Auto Accounting

.....
.....    nfsstat -m
.....


.....
.....    nfsstat -n
.....


Server nfs:
calls      badcalls   public_v2  public_v3  
0          0          0          0          
Version 2: (0 calls)
null       getattr    setattr    root       lookup     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
wrcache    write      create     remove     rename     link       symlink    
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
mkdir      rmdir      readdir    statfs     
0 0%       0 0%       0 0%       0 0%       
Version 3: (0 calls)
null       getattr    setattr    lookup     access     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
write      create     mkdir      symlink    mknod      remove     rmdir      
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
rename     link       readdir    readdir+   fsstat     fsinfo     pathconf   
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
commit     
0 0%       

Client nfs:
calls      badcalls   clgets     cltoomany  
0          0          0          0          
Version 2: (0 calls)
null       getattr    setattr    root       lookup     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
wrcache    write      create     remove     rename     link       symlink    
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
mkdir      rmdir      readdir    statfs     
0 0%       0 0%       0 0%       0 0%       
Version 3: (0 calls)
null       getattr    setattr    lookup     access     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
write      create     mkdir      symlink    mknod      remove     rmdir      
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
rename     link       readdir    readdir+   fsstat     fsinfo     pathconf   
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
commit     
0 0%       

.....
.....    nfsstat -r
.....


Server rpc:
Connection oriented:
calls      badcalls   nullrecv   badlen     xdrcall    dupchecks  dupreqs    
1          0          0          0          0          0          0          
Connectionless:
calls      badcalls   nullrecv   badlen     xdrcall    dupchecks  dupreqs    
0          0          0          0          0          0          0          

Client rpc:
Connection oriented:
calls      badcalls   badxids    timeouts   newcreds   badverfs   timers     
0          0          0          0          0          0          0          
nomem      cantconn   interrupts 
0          0          0          
Connectionless:
calls      badcalls   retrans    badxids    timeouts   newcreds   badverfs   
12         0          0          0          0          0          0          
timers     nomem      cantsend   
0          0          0          

.....
.....    nfsstat -s
.....


Server rpc:
Connection oriented:
calls      badcalls   nullrecv   badlen     xdrcall    dupchecks  dupreqs    
1          0          0          0          0          0          0          
Connectionless:
calls      badcalls   nullrecv   badlen     xdrcall    dupchecks  dupreqs    
0          0          0          0          0          0          0          

Server nfs:
calls      badcalls   public_v2  public_v3  
0          0          0          0          
Version 2: (0 calls)
null       getattr    setattr    root       lookup     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
wrcache    write      create     remove     rename     link       symlink    
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
mkdir      rmdir      readdir    statfs     
0 0%       0 0%       0 0%       0 0%       
Version 3: (0 calls)
null       getattr    setattr    lookup     access     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
write      create     mkdir      symlink    mknod      remove     rmdir      
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
rename     link       readdir    readdir+   fsstat     fsinfo     pathconf   
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
commit     
0 0%       

.....
.....    nfsstat -c
.....


Client rpc:
Connection oriented:
calls      badcalls   badxids    timeouts   newcreds   badverfs   timers     
0          0          0          0          0          0          0          
nomem      cantconn   interrupts 
0          0          0          
Connectionless:
calls      badcalls   retrans    badxids    timeouts   newcreds   badverfs   
12         0          0          0          0          0          0          
timers     nomem      cantsend   
0          0          0          

Client nfs:
calls      badcalls   clgets     cltoomany  
0          0          0          0          
Version 2: (0 calls)
null       getattr    setattr    root       lookup     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
wrcache    write      create     remove     rename     link       symlink    
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
mkdir      rmdir      readdir    statfs     
0 0%       0 0%       0 0%       0 0%       
Version 3: (0 calls)
null       getattr    setattr    lookup     access     readlink   read       
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
write      create     mkdir      symlink    mknod      remove     rmdir      
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
rename     link       readdir    readdir+   fsstat     fsinfo     pathconf   
0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       0 0%       
commit     
0 0%       

.....
.....    nfso -a
.....

           client_delegation = 1
           nfs_max_read_size = 65536
          nfs_max_write_size = 65536
                 nfs_rfc1323 = 1
   nfs_securenfs_authtimeout = 0
    nfs_server_base_priority = 0
           nfs_server_clread = 1
      nfs_use_reserved_ports = 1
   nfs_v3_server_readdirplus = 1
    nfs_v4_fail_over_timeout = 0
                   portcheck = 0
           server_delegation = 1
             utf8_validation = 1
